src/main/java/org/spongepowered/common/mixin/core/command/MixinCommandBase.java

```diff
@@ -43,4 +50,15 @@ public void setExpandedSelector(boolean expandedSelector) {
         this.expandedSelector = expandedSelector;
     }
 
+    @Override
+    public void updateNamespacedAlias(String ownerId) {
+        this.namespacedAlias = ownerId + ":" + getName();
+    }
+
+    @Redirect(method = "checkPermission(Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/command/ICommandSender;)Z",
```

Comment:
Is the full method descriptor necessary here? Usually you can omit the `(Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/command/ICommandSender;)Z` if only one method with the same name exists in the target class (which is the case here I think?).